creator-node/src/routes/users.js

```diff
@@ -145,4 +153,21 @@ module.exports = function (app) {
     await sessionManager.deleteSession(req.get(sessionManager.sessionTokenHeader))
     return successResponse()
   }))
+
+  /**
+   * Returns latest clock value stored in CNodeUsers entry given wallet, or -1 if no entry found
+   */
+  app.get('/users/clock_status/:walletPublicKey', handleResponse(async (req, res) => {
```

Comment:
TODO - add TODO comment to modify to accept wallets array for easy cross-cnode querying